src/main/java/com/saucelabs/ci/sauceconnect/AbstractSauceTunnelManager.java

```diff
@@ -533,6 +534,11 @@ public Process openConnection(
       final Process process =
           prepAndCreateProcess(
               username, apiKey, port, sauceConnectJar, options, printStream, sauceConnectPath);
+
+      // Print sauceconnect process stdout/stderr
+      new ProcessOutputPrinter(process.getInputStream(), (String x) -> logMessage(printStream, x)).start();
```

Comment:
`getInputStream`? I'd expect Output and Error as for stderr/stdout.

Reply:
For some reason, a process InputStream represents stdout, and OutputStream represents process input
* https://docs.oracle.com/javase/9/docs/api/java/lang/Process.html#getInputStream--

<img width="855" alt="image" src="https://github.com/user-attachments/assets/693c9554-40ae-41ac-8a3f-21844ca99041" />

Reply:
I see. The matter of one's POV 🤔

Reply:
Still odd.